README.md

```diff
@@ -189,27 +191,105 @@ You can read and modify the configurations in `config-*.ini` files, accordingly:
+  - `DistThreshold`: Configures the distance threshold for the *social distancing violations*
+  - `Anonymize`: A boolean parameter to enable/disable anonymization of faces in videos and screenshots.
+
+- `[Logger]`:
+  - `Name`: Configures the chosen logger. For now, we only support the *csv_logger*.
+  - `TimeInterval`: Sets the desired logging interval for objects detections and violations.
+  - `LogDirectory`: Defines the location where the generated files will be stored.
+  - `EnableReports`: A boolean parameter to enable/disable the reports generation.
+  - `HeatmapResolution`: Sets the resolution used by the heatmap report.
 
 ### API usage
 After you run the processor on your node, you can use the exposed API to control the Processor's Core, where all the process is getting done.
 
-* Some of the API supported paths are now as the following:
-
- 1- `PUT PROCESSOR_IP:PROCESSOR_PORT/start-process-video`: Sends command `PROCESS_VIDEO_CFG` to core and returns the response. It starts to process the video adressed in the configuration file. If the response is `true`, it means the core is going to try to process the video (no guarantee if it will do it), and if the response is `false`, it means the process can not be started now (e.g. another process is already requested and running)
-
- 2- `PUT PROCESSOR_IP:PROCESSOR_PORT/stop-process-video`: Sends command `STOP_PROCESS_VIDEO` to core and returns the response. It stops processing the video at hand, returns the response `true` if it stopped or `false`, meaning it can not (e.g. no video is already being processed to stop!)
-
- 3- `GET PROCESSOR_IP:PROCESSOR_PORT/config`: It returns the config which is used by processor in json format. This is the file you have used in your Processor's Dockerfile.
-
- 4- `PUT PROCESSOR_IP:PROCESSOR_PORT/config`: It sets the given set of json configurations in the config, and reloads the configuration. Core's engine is also restarted so all methods and members (specially those which were initiated with the old config) can use the updated config (this will stop the processing of the video - if any).
+The available endpoints are grouped in the following subapis:
+- `/config`: provides a pair of endpoint to retrieve and overwrite the current configuration file.
+- `/cameras`: provides endpoints to execute all the CRUD operations required by cameras. This endpoint sare very useful to edit the camera's configuration without restarting the docker process. Additionally, this subapi exposes some calibration endpoints.
+- `/areas`: provides endpoints to execute all the CRUD operations required by areas.
+- `/reports`: a set of endpoints to retrieve the data generated by the processor.
+- `/slack`: a set of endpoints required to configure slack correctly in the processor. We recommend use these endpoints from the [UI](https://beta.lanthorn.ai)instead of calling them directly.
```

Comment:
```suggestion
- `/slack`: a set of endpoints required to configure Slack correctly in the processor. We recommend to use these endpoints from the [UI](https://beta.lanthorn.ai) instead of calling them directly.
```